lib/db/mysql/patches/patch-001-002.sql

```diff
@@ -0,0 +1,30 @@
+--  Create the initial set of tables.
+--
+--  Since this is the first migration, we use `IF NOT EXISTS` to allow us
+--  to run this on a db that already has the original schema in place. The
+--  patch will then be a no-op.  Subsequent patches should *not* use `IF
+--  NOT EXISTS` but should fail noisily if the db is in an unexpected state.
+
+CREATE TABLE IF NOT EXISTS avatar_providers (
+  id INT AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Since we're grafting the mysql-patcher into the workflow here, the `IF NOT EXISTS` is fine here (ditto for the next two tables too). For reference, for future patches we wouldn't use it since we want each patch to be deterministic. Either it all worked, or something was wrong. So yeah, this is okay for this patch.